srcs/Transaction/FileReadExecutor.cpp

```diff
@@ -15,9 +23,36 @@ FileReadExecutor &FileReadExecutor::operator=(FileReadExecutor const &other) {
 
 FileReadExecutor::~FileReadExecutor() {}
 
+HTTPResponse *FileReadExecutor::GetFileExec(std::string file_path) {
+    std::ifstream ifs(file_path.c_str());
+    std::string str;
+    std::ostringstream oss;
+
+    while (getline(ifs, str)) {
+        oss << str << std::endl;
+    }
+    logging_.Debug("file read finished");
+    logging_.Debug(oss.str());
+    return ResponseBuilder::Build(oss.str());
```

Comment:
```suggestion
    std::ifstream ifs(file_path.c_str());
    std::ostringstream oss;

    oss << ifs.rdbuf();

    logging_.Debug("file read finished");
    logging_.Debug(oss.str());
    return ResponseBuilder::Build(oss.str());
```

こっちだとおそらく一気に読み込んでくれます

Reply:
あざます！修正しました！